src/com/ebcitron/piggybank/Main.java

```diff
@@ -0,0 +1,19 @@
+package com.ebcitron.piggybank;
+
+import javax.swing.*;
+
+public class Main {
+
+    public static void main(String[] args) {
```

Comment:
There is some sample data in the readme file, I would suggest that you try to use it here to show that all of your classes are working